1.0/docs/devguide/properties.md

```diff
@@ -422,70 +422,42 @@ Example:
       </script>
     </dom-module>
 
-### Deep sub-property observation {#deep-observation}
-
-To call an observer when any (deep) sub-property of an
-object changes, specify a path with a wildcard (`*`).
-
-When you specify a path with a wildcard, the argument passed to your
-observer is a change record object with the following properties:
+### Observe array mutations
 
-*   `path`. Path to the property that changed. 
-*   `value`. New value of the path that changed.
-*   `base`. The object matching the non-wildcard portion of the path. 
+Use a splice observer to call a method whenever an array item is added
```

Comment:
I would go with "array mutation observer" as the generic term, as used in the heading.